### DISCUSSION TITLE
Problem install Jep python library

### DISCUSSION THREAD

**Discussion by fcscoutinho** [sepinf-inc/IPED#453] [Q&A]
I am trying to install the Jep library to use with iped with python. My version of Java is 251.
When trying to install you receive the following error message: jep unable to find vcvarsall bat setup python setup.py build
How can I fix this problem and put iped to use python properly?

**Comment by lfcnassif** [comment]
I have updated the docs about JEP, see if that helps: https://github.com/sepinf-inc/IPED/wiki/User-Manual#python-modules

**Comment by lfcnassif** [comment]
Just updated the docs again: https://github.com/sepinf-inc/IPED/wiki/User-Manual#python-modules

(I think you need to install MS Visual C++ 2015 Build Tools)
